src/patients/GeneralInformation.tsx

```diff
@@ -148,7 +140,14 @@ const GeneralInformation = (props: Props) => {
                 type="number"
                 value={`${differenceInYears(new Date(Date.now()), new Date(patient.dateOfBirth))}`}
                 isEditable={isEditable}
-                onChange={onApproximateAgeChange}
+                onChange={
+                  (event) =>
+                    onFieldChange(
+                      'dateOfBirth',
+                      guessDOBfromApproximateAge(event.currentTarget.value),
+                    )
+                  // eslint-disable-next-line react/jsx-curly-newline
```

Comment:
let's remove the eslint disable, and format to however eslint perferes.

Reply:
It seems like ESLint doesn't have a solution for a case like this? We are unable to proceed without `eslint-disable-next-line`.

Reply:
✅ This is due to a conflict between eslint and prettier. Proceeding by extracting what's in the bracket into a constant.